### ISSUE TITLE
Add visual notification on new chat message

### ISSUE THREAD

**Issue by delthas** [pdaddyo/soundbounce#136]
It'd be nice if there was a way to be able to know when someone sends a message on a room. I think sound notifications might get boring real quick especially in such a software, what about a visual notification? Maybe as an optional feature, so as not to bother "listen only" guys?
##

**Comment by pdaddyo** [comment]
Great idea, this would be v useful.  Even a simple ' * ' in the window title to show unread messages (Slack uses this to great effect).  Thanks for your suggestion.

**Comment by Allikin** [comment]
I wanna know if someone chats when I'm tabbed out. Is it possible to let the menu bar button glow, like the Skype one does?

**Comment by pdaddyo** [comment]
Yep. For windows I'll use this technique http://whiledeveloping.wikidot.com/flashing-taskbar

**Comment by pdaddyo** [comment]
Just realised this is a dupe of #66 - closing the earlier one since this has more chat in.
